### PR TITLE
ref: Update android wizard with latest SAGP version and new File I/O integration

### PR DESCRIPTION
**Pull request by romtsn**
Has to be merged after #4575 and #4574

### CODE REVIEW

File: src/wizard/android/index.md [hunk 1]
```diff
@@ -116,9 +116,9 @@ transaction.finish() // Mark the transaction as finished and send it to Sentry
 
 Check out [the documentation](https://docs.sentry.io/platforms/android/performance/instrumentation/) to learn more about the API and automatic instrumentations.
 
-> Want to play with some new features? Try out our beta [Room performance integration](https://docs.sentry.io/platforms/android/performance/instrumentation/automatic-instrumentation/#sqlite-and-room-instrumentation).
+> Want to play with some new features? Try out our beta [Room](https://docs.sentry.io/platforms/android/configuration/integrations/room-and-sqlite/) and [File I/O](https://docs.sentry.io/platforms/android/configuration/integrations/file-io/) performance integratins.
```
Comment (romtsn) [inline src/wizard/android/index.md:119]:
```suggestion
> Want to play with some new features? Try out our beta [Room](https://docs.sentry.io/platforms/android/configuration/integrations/room-and-sqlite/) and [File I/O](https://docs.sentry.io/platforms/android/configuration/integrations/file-io/) performance integrations.
```

File: src/wizard/android/index.md [hunk 2]
```diff
@@ -116,9 +116,9 @@ transaction.finish() // Mark the transaction as finished and send it to Sentry
...
+> Want to play with some new features? Try out our beta [Room](https://docs.sentry.io/platforms/android/configuration/integrations/room-and-sqlite/) and [File I/O](https://docs.sentry.io/platforms/android/configuration/integrations/file-io/) performance integrations.
 >  
-> This feature is available in the Beta release of the [Sentry Android Gradle plugin](https://docs.sentry.io/platforms/android/gradle); you must use version `3.0.0-beta.1`. The `tracingInstrumentation` option is enabled by default, so Sentry automatically measures performance of the database queries done with Room if you set a tracing sample rate. Features in Beta are still a work-in-progress and may have bugs. We recognize the irony.
+> This feature is available in the Beta release of the [Sentry Android Gradle plugin](https://docs.sentry.io/platforms/android/gradle); you must use version `3.0.0-beta.3`. The `tracingInstrumentation` option is enabled by default, so Sentry automatically measures performance of the database queries done with Room as well as File I/O operations if you set a tracing sample rate. Features in Beta are still a work-in-progress and may have bugs. We recognize the irony.
```
Comment (philipphofmann) [inline src/wizard/android/index.md:121]:
```suggestion
> This feature is available in the Beta release of the [Sentry Android Gradle plugin](https://docs.sentry.io/platforms/android/gradle); you must use version `3.0.0-beta.3`. The `tracingInstrumentation` option is enabled by default, so Sentry automatically measures the performance of the database queries done with Room as well as file I/O operations if you set a tracing sample rate. Features in Beta are still a work-in-progress and may have bugs. We recognize the irony.
```

**Review by philipphofmann — APPROVED** [review]
One minor suggestion.

File: src/wizard/android/index.md [hunk 3]
```diff
@@ -116,9 +116,9 @@ transaction.finish() // Mark the transaction as finished and send it to Sentry
...
+> Want to play with some new features? Try out our beta [Room](https://docs.sentry.io/platforms/android/configuration/integrations/room-and-sqlite/) and [File I/O](https://docs.sentry.io/platforms/android/configuration/integrations/file-io/) performance integrations.
 >  
-> This feature is available in the Beta release of the [Sentry Android Gradle plugin](https://docs.sentry.io/platforms/android/gradle); you must use version `3.0.0-beta.1`. The `tracingInstrumentation` option is enabled by default, so Sentry automatically measures performance of the database queries done with Room if you set a tracing sample rate. Features in Beta are still a work-in-progress and may have bugs. We recognize the irony.
+> This feature is available in the Beta release of the [Sentry Android Gradle plugin](https://docs.sentry.io/platforms/android/gradle); you must use version `3.0.0-beta.3`. The `tracingInstrumentation` option is enabled by default, so Sentry automatically measures the performance of the database queries done with Room as well as file I/O operations if you set a tracing sample rate. Features in Beta are still a work-in-progress and may have bugs. We recognize the irony.
```
Comment (imatwawana) [inline src/wizard/android/index.md:121]: Should the "F" in "file I/O" be capitalized here?
```suggestion
> This feature is available in the Beta release of the [Sentry Android Gradle plugin](https://docs.sentry.io/platforms/android/gradle); you must use version `3.0.0-beta.3`. The `tracingInstrumentation` option is enabled by default, so Sentry automatically measures the performance of the database queries done with Room as well as file I/O operations if you set a tracing sample rate. Features in Beta are still a work-in-progress and may have bugs. We recognize the irony.
```

**Review by imatwawana — APPROVED** [review]
One style question, but otherwise looks good!

File: src/wizard/android/index.md (hunk 3 above)
Comment (romtsn) [reply src/wizard/android/index.md:121]: haha, that's exactly what @philipphofmann has changed, but I also think it should be capitalized here

Comment (imatwawana) [reply src/wizard/android/index.md:121]: Ha! Ultimately, it just needs to be consistent. It's capitalized throughout the File I/O documentation and always capitalize all the other integration names, so I'm not sure why we wouldn't do that for this one.

**Pull request merged**